### ISSUE TITLE
Navigation Block: Empty state isn't visible against dark background

### ISSUE THREAD

**Issue by annezazu** [WordPress/gutenberg#29235]
## Description

The navigation block's default state isn't visible against a dark background. 

## Step-by-step reproduction instructions

Please list the steps needed to reproduce the bug. For example:
1. Open the site editor or a new page/post. 
2. Add a cover block and set the background to a dark color like black. 
3. Add a navigation block.
4. Select "Start Empty" and see that the default state is almost impossible to see. 

## Expected behaviour

I expected to be able to see the default state. 

## Actual behaviour

I'm unable to see the default, empty state. 

## Screenshots or screen recording (optional)

Screenshot:

<img width="470" alt="Screen Shot 2021-02-22 at 3 17 41 PM" src="https://user-images.githubusercontent.com/26996883/108777505-20d2f980-7521-11eb-9552-168dec273e18.png">

Video: 

https://user-images.githubusercontent.com/26996883/108777432-06991b80-7521-11eb-826d-4ec770969ced.mov

## WordPress information
- WordPress version: 5.6.2
- Gutenberg version: 10.1
- Are all plugins except Gutenberg deactivated? Yes
- Are you using a default theme (e.g. Twenty Twenty-One)? TT1 Blocks
